src/nft/components/explore/ValueProp.tsx

```diff
@@ -0,0 +1,33 @@
+import clsx from 'clsx'
+import { Box } from 'nft/components/Box'
+import { Column } from 'nft/components/Flex'
+
+import MarketplacesImage from '../../../assets/images/nft-marketplaces.png'
+import * as styles from './Explore.css'
+
+const ValueProp = () => {
+  return (
+    <Box width="full">
+      <Column as="section" className={styles.section}>
+        <Box
+          className={clsx(styles.bannerWrap, styles.valuePropWrap)}
+          style={{
+            height: '135px',
+            background: `url(${MarketplacesImage}) top 0 right 0 no-repeat`,
```

Comment:
When the banner reduces for smaller viewports, I think we should be left instead of right aligned, this way as the screen shrinks, the more well known marketplaces stay visible